native/src/routes/landing/components/CitySelector.js

```diff
@@ -60,8 +60,6 @@ class CitySelector extends React.PureComponent<PropsType> {
     const cities = this.props.cities
     if (normalizedFilter === 'wirschaffendas') {
       return cities.filter(_city => !_city.live)
-    } else if (buildConfig().development) {
```

Comment:
Should we add a new flag for this? Not sure how to handle this. I think we can remove it actually.

Not sure how to check whether metro is running as server or whether `react-native bundle` has been called.

Reply:
I think it was nice to have this, but I guess it isn't a real problem if the effort is to high as you can just enter 'wirschaffendas' as well. I really wouldn't mind just using the logging flag with a comment here as well... :innocent: I mean, we just log all cities... to the screen... visible to the user... :grin:

Reply:
I also think this was a quite nice feature, but I agree with @klinzo here. No need to put in too much effort.